src/main/java/com/saralein/server/Constants.java

```diff
@@ -7,6 +7,12 @@
 public final class Constants {
```

Comment:
I am skeptical of `Constants`. It feels weird to have command-line argument constants next to request parsing constants. 🤔

I am not suggesting you change it, just expressing my skepticism 😄